public/js/debugResults.js

```diff
@@ -293,6 +297,63 @@
 	var alchemy = new Alchemy(config)
     }
 
+    function showTable(alchemy_data) {
+	TableDiv.innerHTML = ""
+
+	piv.html(TableDiv, "p", "Each would-be victory in a head-to-head matchup is represented by a \
+number in the following table.  Start with a candidate along the \
+vertical access.  The numbers in that row indicate the strengths of \
+the victories that the candidate would have in head-to-head matchups \
+against each candidate along the horizontal access.")
+	
+	var table = piv.html(TableDiv, "table", "")
+	// TODO: move this to a style sheet
+	table.style.borderCollapse = "collapse"
+	table.style.textAlign = "center"
+	table.style.vertialAlign = "middle"
+	
+	var nodes = alchemy_data.nodes.map(node => node.id).sort()
```

Comment:
Does that new => lambda syntax break in not-that-old browsers (can't test, on phone)?

Reply:
Browser compatibility hasn't been something on my radar.  Is there a good way to check these things?  (e.g., ideally a tool that tells us the level of support for the features used in a given page).  I think this was added in ES 6: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions.  Do we consider these things based on ES level, or choose specific browsers we care about?

Reply:
Good question. I don't know of such a tool.

Previous front-end projects I was on all established a minimum compatibility target (example IE 9) and then catered to that browser. It's been a yearish since I've done front-end web development and it's possible the practices have changed.

If we were to pick IE11, we could not use several ES 6 features:
https://kangax.github.io/compat-table/es6/#ie11

It looks like IE is still being pretty widely used:
[netmarketshare](https://netmarketshare.com/browser-market-share.aspx?options=%7B%22filter%22%3A%7B%22%24and%22%3A%5B%7B%22deviceType%22%3A%7B%22%24in%22%3A%5B%22Desktop%2Flaptop%22%5D%7D%7D%5D%7D%2C%22dateLabel%22%3A%22Trend%22%2C%22attributes%22%3A%22share%22%2C%22group%22%3A%22browser%22%2C%22sort%22%3A%7B%22share%22%3A-1%7D%2C%22id%22%3A%22browsersDesktop%22%2C%22dateInterval%22%3A%22Monthly%22%2C%22dateStart%22%3A%222017-07%22%2C%22dateEnd%22%3A%222018-06%22%2C%22segments%22%3A%22-1000%22%7D)
[GlobalStats](http://gs.statcounter.com/browser-market-share/desktop/worldwide)

Some relevant questions that I can't answer, just food for thought:
How likely is our user group to use old incompatible browsers? What is the impact of incompatibility on user experience? What is the benefit of using the ES6 features?